paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -126,9 +127,10 @@ def issue_state_change_for_service(service_config, force_bounce, desired_state):
         force_bounce=force_bounce,
         desired_state=desired_state,
     )
-    remote_git.create_remote_refs(
-        utils.get_git_url(service_config.get_service()), ref_mutator
```

Comment:
Is it enough for this to just be called on stop/stop/restart? We often make use of paasta mark-for-deployment alone in Jenkinsfiles -- is paasta restart always called by paasta itself when a sha is marked for deployment?

This PR might break `def test_mark_for_deployment_yelpy_repo`

Reply:
It is still called from mark-for-deployment -- it is just not defined in that module anymore (I think this is what you are asking?)

Reply:
Ah I misread the diff, looks good to me then! Ship!